### PR TITLE
Added Spanish translation for quickstart.md

### PR DESCRIPTION
**Pull request by victormartinezconte**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

## Quick checklist

The following checklist helps ensure your PR is complete:

- [ ] Reviewed the [Contributing Guidelines](https://github.com/deeplearning4j/deeplearning4j/blob/master/CONTRIBUTING.md) and followed the steps within.
- [ ] Created tests for any significant new code additions.
- [ ] Relevant tests for your changes are passing.
- [ ] Ran mvn formatter:format (see [formatter instructions](http://code.revelc.net/formatter-maven-plugin/examples.html#Setting_Source_Files) for targeting your specific files).

### CODE REVIEW

**Comment by agibsonccc** [comment]
Thanks for submitting this. We will take a bit of time to consolidate this. Overall it looks like you're doing things well enough. Eg, the folder layouts and the like.

**Comment by victormartinezconte** [comment]
Thank you for your comments, much appreciated!

File: es/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+setTimeout(ldinsp, 500); document.readyState != "complete" ? (window.attachEvent ? window.attachEvent('onload', ldinsp) : window.addEventListener('load', ldinsp, false)) : ldinsp();
+})();
+</script>
+<!-- End Inspectlet Embed Code -->
+
+# Gu&iacute;a de inicio r&aacute;pido
+
+Esto es todo lo que tiene que saber para ejecutar los ejemplos de DL4J y empezar sus propios proyectos.
+
+Le recomendamos unirse a nuestro [Chat en vivo de Gitter](https://gitter.im/deeplearning4j/deeplearning4j). Gitter es el sitio donde puede pedir ayuda y hacer comentarios, pero por favor use esta gu&iacute;a antes de hacer preguntas que ya hemos respondido anteriormente. Si es nuevo en deep learning, hemos incluido [una hoja de ruta para principiantes](../deeplearningforbeginners.html) con enlaces a cursos, lecturas y otros recursos. Si necestia un tutorial de principio a fin para comenzar (incluida la configuraci&oacute;n), por favor vaya a nuestra p&aacute;gina [primeros pasos](http://deeplearning4j.org/gettingstarted).
```
Comment (huitseeker) [inline es/quickstart.md]: Tal vez quiere usted acordar las mayúsculas de "Chat en vivo de Gitter" con el texto. Aqui: "chat".

File: es/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+* [Maven In Five Minutes](http://maven.apache.org/guides/getting-started/maven-in-five-minutes.html)
+
+#### <a name="IntelliJ">IntelliJ IDEA</a>
+
+Un Entorno de Desarrollo Integrado ([IDE en ingl&eacute;s](http://encyclopedia.thefreedictionary.com/integrated+development+environment)) le permite trabajar con nuestra API y configurar redes neuronales en unos pocos pasos. Le recomendamos encarecidamente usar [IntelliJ](https://www.jetbrains.com/idea/download/), que se integra con Maven para gestionar las dependencias. La [community edition de IntelliJ](https://www.jetbrains.com/idea/download/) es gratuita. 
+
+Hay otros IDEs populares como [Eclipse](https://books.sonatype.com/m2eclipse-book/reference/creating-sect-importing-projects.html) y [Netbeans](http://wiki.netbeans.org/MavenBestPractices). IntelliJ es preferido, y utiliz&aacute;ndolo har&aacute; m&aacute;s sencillo encontrar ayuda en el [Chat en vivo de Gitter](https://gitter.im/deeplearning4j/deeplearning4j) si lo necesita.
```
Comment (huitseeker) [inline es/quickstart.md]: Como arriba.

File: es/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+  * [Early Adopters](https://gitter.im/deeplearning4j/deeplearning4j/earlyadopters) es para aquellos que nos est&aacute;n ayudando en la revisi&oacute;n y mejorando la siguiente entrega. ATENCI&Oacute;N: Este es para la gente m&aacute;s experimentada.
+2. Lea la [introducción a las redes neuronales profundas](./neuralnet-overview) o [uno de nuestros detallados tutoriales](./tutorials).
+3. Eche un vistazo a la m&aacute;s detallada [Gu&iacute;a Extensa de Configuraci&oacute;n](./gettingstarted).Check out the more detailed [Comprehensive Setup Guide](./gettingstarted).
+4. Explore la [documentaci&oacute;n de DL4J](./documentation).
+5. **Gente de Python**: si tiene planeado evaluar el rendimiento de Deeplearning4j para compararlo con otro bien conocido framework de Python [x], por favor lea [estas instrucciones](https://deeplearning4j.org/benchmark) sobre c&oacute;mo optimizar el tama&ntilde;o del heap, el recolector de basura y ETL en la JVM. Sigui&eacute;ndolos, observar&aacute; una mejora de al menos *10x de velocidad en el tiempo de entrenamiento*.
```
Comment (huitseeker) [inline es/quickstart.md]: Esta caja (`[x]`) es problematica.

Comment (victormartinezconte) [reply es/quickstart.md]: Hi @huitseeker, the `[x]` box is also present in the original (english) quickstart.md file, should I remove it?.

**Pull request merged**